### PR TITLE
Remove superfluous uses of 'as' casting.

### PR DESCRIPTION
**Pull request by kberg**

### CODE REVIEW

File: src/cards/promo/Splice.ts [hunk 1]
```diff
@@ -48,7 +47,15 @@ export class Splice extends Card implements CorporationCard {
     return undefined;
   }
 
-  public onCardPlayed(player: Player, card: IProjectCard | CorporationCard) {
+  public onCardPlayed(player: Player, card: IProjectCard) {
+    returnthis._onCardPlayed(player, card);
```
Comment (bafolts) [inline src/cards/promo/Splice.ts:51]: This should be `return this`.

**Review by bafolts — APPROVED** [review]
Save "as" for similes.

File: src/cards/promo/Splice.ts (hunk 1 above)
Comment (kberg) [reply src/cards/promo/Splice.ts]: Done. Gah. Feel free to make an edit yourself next time if you prefer.

**Comment by kberg** [comment]
> Save "as" for similes.

chuckle

**Pull request merged**